src/coreclr/jit/ee_il_dll.cpp

```diff
@@ -437,16 +437,14 @@ unsigned Compiler::eeGetArgSize(CORINFO_ARG_LIST_HANDLE list, CORINFO_SIG_INFO*
                 }
             }
         }
-#elif defined(TARGET_LOONGARCH64)
+//  otherwise will we pass this struct by value in multiple registers
+#elif !defined(TARGET_ARM)
         // Any structs that are larger than MAX_PASS_MULTIREG_BYTES are always passed by reference
         if (structSize > MAX_PASS_MULTIREG_BYTES)
         {
             // This struct is passed by reference using a single 'slot'
             return TARGET_POINTER_SIZE;
         }
-//  otherwise will we pass this struct by value in multiple registers
-#elif !defined(TARGET_ARM)
-        NYI("unknown target");
```

Comment:
Can you keep this NYI and expand the earlier case instead to what fits RISC-V's calling convention?

Reply:
Thank you!